onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1578,45 +1578,10 @@ Matrix product that behaves like numpy.matmul: https://docs.scipy.org/doc/numpy-
         ONNX_NAMESPACE::matmulShapeInference(ctx, 0, 1);
       });
 
-  static const char* TransposeMatMul_doc = R"DOC(
-Duplicate of FusedMatMul. Going forward FusedMatMul should be used. This OP will be supported for backward compatibility.
```

Comment:
As far as I know, this op can only be introduced during graph transformation. 
Since we switch to use FuseMatmul in the graph transformer, I don't think there will be other models depending on this op.

Reply:
we do allow model serialization after graph transformation therefore there is a chance there are models out there which have this operator. We decided to keep this around to be on the safer side since we promise backward compatibility. 
Other than code clean up is there any other motivation for this change? The binary size increase due to this should be minimal... we decided to make the tradeoff to make sure backward compatibility is maintained.

Reply:
We will keep TransposeMatMul in the schema for backward compatibility